packages/traceloop-sdk/pyproject.toml

```diff
@@ -52,6 +52,7 @@ opentelemetry-instrumentation-llamaindex = { path = "../opentelemetry-instrument
 opentelemetry-instrumentation-milvus = { path = "../opentelemetry-instrumentation-milvus", develop = true }
 opentelemetry-instrumentation-haystack = { path = "../opentelemetry-instrumentation-haystack", develop = true }
 opentelemetry-instrumentation-bedrock = { path = "../opentelemetry-instrumentation-bedrock", develop = true }
+opentelemetry-instrumentation-redis = { path = "../opentelemetry-instrumentation-redis", develop = true }
```

Comment:
This doesn't exist - it's an otel package (like opentelemetry-instrumentation-urllib3). You also need to run `poetry lock`